src/ray/protobuf/pubsub.proto

```diff
@@ -43,3 +55,53 @@ message WorkerRefRemovedMessage {
   // still be borrowing an object ID that was nested inside.
   repeated ObjectReferenceCount borrowed_refs = 1;
 }
+
+///
+/// Subscribe
+///
+message Command {
```

Comment:
Should we just call it the Message?  I feel it's more common to deliver messages by pub/sub but not commands.

Reply:
What's the common term in other pubsub systems?

Reply:
Message seems to be a little generic here.

Reply:
I'm OK with command here with comment.